### PR TITLE
add UseDeviceAuthentication

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/11190

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 1]
```diff
@@ -508,6 +509,20 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -UseDeviceAuthentication
+Use device code authentication instead of a browser control
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:513]:
```suggestion
Use device code authentication instead of a browser control.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merged this PR. Thanks

**Pull request merged**